### PR TITLE
Feature/common style sheets

### PR DESCRIPTION
**Pull request by magovansh9**
Created new StyleSheets for jsx files with overlapping css.

### CODE REVIEW

**Comment by magovansh9** [comment]
The naming of the stylesheet jsx files can be improved, please suggest appropriate names.

File: app/src/src/stylesheets/views/views.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+
+/* Used in AdView, EventView, ListingView, PollView */
```
Comment (deandrebaker) [inline app/src/src/stylesheets/views/views.jsx:2]: We can delete these 2 lines.

File: app/src/src/stylesheets/views/views.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import { StyleSheet } from "react-native";
+
```
Comment (deandrebaker) [inline app/src/src/stylesheets/views/views.jsx:4]: You can add a comment under this line saying what this stylesheet is for (e.g. Styles for Shared Item views)

File: app/src/src/stylesheets/views/views.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+const commonViewStyles = StyleSheet.create({
```
Comment (deandrebaker) [inline app/src/src/stylesheets/views/views.jsx:5]: We can add 'export' infront of const here to save lines.
`export const commonViewStyles = StyleSheet.create({`

Also, we should rename this constant to something like 'sharedItemViewStyles'

File: app/src/src/stylesheets/views/views.jsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    },
+    bold: {
+      fontWeight: 'bold',
+    },
+    date: {
+      color: 'grey',
+    },
+  });
+
+  export {commonViewStyles}
```
Comment (deandrebaker) [inline app/src/src/stylesheets/views/views.jsx:48]: We can delete this line

File: app/src/src/stylesheets/tabs/tabs.jsx [hunk 5]
```diff
@@ -0,0 +1,6 @@
+
+container {
+      flex: 1;
+      background-color: '#eeeeee'
+    }
+
```
Comment (deandrebaker) [inline app/src/src/stylesheets/tabs/tabs.jsx:6]: Do we need this file? Nothing here is exported.

File: app/src/src/screens/likes/LikesScreen.jsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+        filtersList={[likeFilters]}
+        scrollRef={ref}
+      />
+      <CreateItemModal navigation={navigation} />
+    </View>
+  );
+};
+
+export default LikesScreen;
+
```
Comment (deandrebaker) [inline app/src/src/screens/likes/LikesScreen.jsx:36]: These changes do not belong in this PR. Please undo this.

File: app/src/src/screens/notifications/NotificationsScreen.jsx [hunk 7]
```diff
@@ -1,6 +1,7 @@
-import { StyleSheet, Text, View } from 'react-native';
+import { Text, View } from 'react-native';
 import Header from '../../components/views/Header';
 import React from 'react';
+import {styles} from '../../stylesheets/messaging/messaging.jsx';
```
Comment (deandrebaker) [inline app/src/src/screens/notifications/NotificationsScreen.jsx:4]: Isn't the messaging.jsx file empty? How can we import 'styles' from it?

File: app/src/src/stylesheets/likesAndComments/likesAndComments.jsx [hunk 8]
```diff
@@ -0,0 +1,12 @@
+/* Used in CommentsScreen, LikedScreen */
+
+import { StyleSheet } from "react-native";
+
+const styles = StyleSheet.create({
```
Comment (deandrebaker) [inline app/src/src/stylesheets/likesAndComments/likesAndComments.jsx:5]: We can use the 'export' keyword infront of 'const' to save lines.

Also, we should give this a more descriptive name than just 'styles'

File: app/src/src/stylesheets/createItem/createItem.jsx [hunk 9]
```diff
@@ -0,0 +1,11 @@
+
+/* Used in CalendarScreen, SavedItemsScreen, SettingsScreen */
+
+import { StyleSheet } from "react-native";
+
+const styles = StyleSheet.create({
```
Comment (deandrebaker) [inline app/src/src/stylesheets/createItem/createItem.jsx:6]: We can use the 'export' keyword infront of 'const' to save lines.

Also, we should give this a more descriptive name than just 'styles'

File: app/src/src/components/views/PostView.jsx [hunk 10]
```diff
@@ -47,45 +48,4 @@ const PostView = (post) => {
...
-    alignSelf: 'flex-start',
-    letterSpacing: 0.5,
-  },
-  bodyText: {
-    marginBottom: 0,
-    letterSpacing: 0.5,
-    fontSize: 15,
-  },
-});
+
```
Comment (deandrebaker) [inline app/src/src/components/views/PostView.jsx:51]: We can delete this line

File: app/src/src/components/views/LikedUserView.jsx [hunk 11]
```diff
@@ -0,0 +1,21 @@
+import { Text, View, StyleSheet, Image } from 'react-native';
+
+const LikedUserView = ({ first_name, profile_photo }) => {
+  return (
+    <>
+      <View style={styles.ListBegin}>
+        <Text style={styles.text}>{first_name}</Text>
+        <Image style={styles.image} source={profile_photo}></Image>
+      </View>
+    </>
```
Comment (deandrebaker) [inline app/src/src/components/views/LikedUserView.jsx:10]: We do not need to wrap this in a react fragment (<></>). We can simply just return the View component

File: app/src/src/components/views/EventView.jsx [hunk 12]
```diff
@@ -128,46 +129,7 @@ const EventView = (event) => {
...
-  title: {
-    alignSelf: 'flex-start',
-    letterSpacing: 0.5,
-  },
-  bodyText: {
-    marginBottom: 0,
-    letterSpacing: 0.5,
-    fontSize: 16,
-  },
+
```
Comment (deandrebaker) [inline app/src/src/components/views/EventView.jsx:132]: We can delete this line.

File: app/src/src/components/views/CommentBar.jsx [hunk 13]
```diff
@@ -60,7 +60,10 @@ const CommentBar = ({ item, contentType, endpoint }) => {
         <View style={styles.likesAndCommentsText}>
           <Text
             onPress={() => {
-              // TODO : Show who has liked this post onPress
+              nav.navigation.navigate(routes.likes, {
+                postId: item.id,
+                contentType: contentType,
+              });
```
Comment (deandrebaker) [inline app/src/src/components/views/CommentBar.jsx:66]: These changes do not belong in this PR. Please undo this.

**Review by deandrebaker — CHANGES_REQUESTED** [review]
@magovansh9 looking really good! Just made some suggestions on things we can clean up.

File: app/src/src/components/views/CommentBar.jsx [hunk 14]
```diff
@@ -58,12 +58,7 @@ const CommentBar = ({ item, contentType, endpoint }) => {
     return (
       <View style={styles.commentBarContainer}>
         <View style={styles.likesAndCommentsText}>
-          <Text
-            onPress={() => {
-              // TODO : Show who has liked this post onPress
-            }}
-            style={styles.likesText}
-          >
+          <Text onPress={() => {}} style={styles.likesText}>
```
Comment (trentstauff) [inline app/src/src/components/views/CommentBar.jsx:61]: Why was this TODO comment removed?

**Review by deandrebaker — APPROVED** [review]

**Pull request merged**